docs/64_dandi_archive.md

```diff
@@ -0,0 +1,219 @@
+
+To do one final test, try using your credentials to log into the Django Admin panel -- it should be located at `/admin` for your API, such as `your-apps-domain.com/admin`
+
+You are all set here!
+
+## Frontend Deployment
+
+A majority of the necessary setup steps here are defined already [during the vendor account setup for Netlify](../60_initialize_vendors/#netlify).
+
+The only other major initial setup step for the DANDI Archive frontend is regarding authentication -- [see here for more details](../61_dandi_authentication/#populating-appropriate-values-for-the-frontend-to-handle-authentication)
+
+## API Deployment
+
+### Via GitHub CI/CD
+
+Within the DANDI Archive repository, GitHub actions workflows exist for deployments to [production](https://github.com/dandi/dandi-archive/blob/master/.github/workflows/backend-production-deploy.yml) and [staging](https://github.com/dandi/dandi-archive/blob/master/.github/workflows/backend-staging-deploy.yml) environments
+
+- **Staging**: by default, releases are manual via the `workflow_dispatch` clause in the workflow
+
+To invoke the pipeline, proceed to the `Actions` tab in GitHub and find the pipeline name in the column on the left
+
+<br/><br/>
+<img
+src="../img/github_actions_workflow.jpg"
+alt="github_actions_workflow"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Proceed to click the `Run Workflow` option.
+
+<br/><br/>
+<img
+src="../img/github_run_workflow.jpg"
+alt="github_run_workflow"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Other than the output of the CI/CD workflow, you can observe the status of the new release via your Heroku app dashboard.
+
+- **Production**:  by default, releases are managed via "tags" applied to given PRs merged into main with the label of "release"
+
+<br/><br/>
+<img
+src="../img/github_label.jpg"
+alt="github_label"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+**Note: `workflow_dispatch` may also be applied if you'd like to be more intentional with releases.**
+
+## Updating Allowed Hosts
+
+For the Django-based DANDI Archive API to receive and send HTTP requests without CORS errors, you'll need to update `ALLOWED_HOSTS` within the `dandiapi/settings.py` file.
+
+The `settings.py` file, in general, can be understood as the configuration file for the Django app -- [see the Django docs for more info here](https://docs.djangoproject.com/en/5.0/topics/settings/)
+
+For the `ALLOWED_HOSTS` value to work, the following code snippet can be added, for example:
+
+```python
+class HerokuProductionConfiguration(DandiMixin, HerokuProductionBaseConfiguration):
+    ...other configurations...
+
+    if 'ALLOWED_HOSTS' not in globals():
+        ALLOWED_HOSTS = []
+
+    ALLOWED_HOSTS += [
+        '<your-api-url>'
+    ]
+```
```

Comment:
Hi @aaronkanzer, maybe something changed in the source code since you wrote these docs but I don't see an `ALLOWED_HOSTS` in [dandiapi/settings.py](https://github.com/dandi/dandi-archive/blob/master/dandiapi/settings.py).  Is this not needed for the main instance - DANDI Archive?